### PR TITLE
sdk-btc & sdk-eth & signature verify

### PR DESCRIPTION
**Pull request by wow-sven**
### feature
Ethereum support personal_sign

### fix 
signature verify
sdk test

### update
fastcrypto dependencies

### remove
rust_secp256k1 dependencies
crypto schnorr 
native_ripemd160
crypto ecdsa_k1_recoverable
Remove okx support

- Closes #issue

### CODE REVIEW

**Review by baichuan3 — APPROVED** [review]

File: sdk/typescript/rooch-sdk-kit/src/feature/index.ts [hunk 1]
```diff
@@ -2,9 +2,44 @@
 // SPDX-License-Identifier: Apache-2.0
 
 export enum SupportWallet {
-  UniSat = 'UniSat',
```
Comment (feliciss) [inline sdk/typescript/rooch-sdk-kit/src/feature/index.ts:5]: So there we unified UniSat and OKX?

File: sdk/typescript/rooch-sdk-kit/src/feature/index.ts [hunk 2]
```diff
@@ -2,9 +2,44 @@
...
+  static get Bitcoin(): BitcoinWallets {
+    return new BitcoinWallets()
+  }
+
+  static get ETH(): ETHSupportWallet {
+    return new ETHSupportWallet()
+  }
+}
+
+class BitcoinWallets {
```
Comment (feliciss) [inline sdk/typescript/rooch-sdk-kit/src/feature/index.ts:23]: Should also support OKX wallet.

File: sdk/typescript/rooch-sdk-kit/src/types/address/bitcoinAddress.ts [hunk 3]
```diff
@@ -3,31 +3,26 @@
...
-  p2pkh = 'p2pkh',
-  p2sh = 'p2sh',
-  p2wpkh = 'p2wpkh',
-  p2wsh = 'p2wsh',
-  p2tr = 'p2tr',
+  p2pkh = 0,
+  p2sh = 1,
+  p2wpkh = 2,
+  p2wsh = 2,
+  p2tr = 2,
```
Comment (feliciss) [inline sdk/typescript/rooch-sdk-kit/src/types/address/bitcoinAddress.ts:23]: We shall call them all as [Bech32](https://en.bitcoin.it/wiki/Bech32) addresses.

File: sdk/typescript/rooch-sdk-kit/src/types/address/multiChainAddress.test.ts [hunk 4]
```diff
@@ -8,17 +8,40 @@ import { MultiChainAddress } from './multiChainAddress'
 
 describe('multiChainAddress', () => {
   it('should parse multi chain address is ok', () => {
-    let expectMultiChainAddress = 'bitcoin1pqrawtj0v3gpl4qqkkcavjdequnj9nswvqu40gx7m'
-    let expectMultiChainAddressBytes = new Uint8Array([
-      0, 0, 0, 0, 0, 0, 0, 0, 22, 2, 0, 250, 229, 201, 236, 138, 3, 250, 128, 22, 182, 58, 201, 55,
-      32, 228, 228, 89, 193, 204, 7,
+    // native swgwit p2wpkh
```
Comment (feliciss) [inline sdk/typescript/rooch-sdk-kit/src/types/address/multiChainAddress.test.ts:11]: swgwit -> Segwit

**Review by feliciss — COMMENTED** [review]
If we were to support `p2tr` protocol, we should keep `schnorr` Rust code. But if you meant to integrate with frontend first, it's fine. We could restore the protocol code later.

File: sdk/typescript/rooch-sdk-kit/src/types/address/bitcoinAddress.ts (hunk 3 above)
Comment (wow-sven) [reply sdk/typescript/rooch-sdk-kit/src/types/address/bitcoinAddress.ts:23]: The method used to distinguish bitcoin address payload type.
It is currently used as a simple tag and is only used in rooch

**Pull request merged**